yt/docker/ya-build/query-tracker/build_system_python_udfs.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# This script builds system python udfs.
+# It is a separate script because it depends on environment(local pythons).
+
+# Required environment variables:
+# YDB_SOURCE_PATH - path to the ydb source root.
+# YQL_BUILD_PATH - path to the build directory. All artifacts will be placed here.
+# BUILD_FLAGS - flags to pass to ya make when building.
+
+set -eux
+shopt -s expand_aliases
+
+export YDB_SOURCE_PATH=$(realpath $YDB_SOURCE_PATH)
+export YQL_BUILD_PATH=$(realpath $YQL_BUILD_PATH)
+
+mkdir -p $YQL_BUILD_PATH
+
+for version in 3.8 \
+               3.9 \
+               3.10 \
+               3.11 \
+               3.12
+do
+  udf_suffix=$(echo "$version" | tr . _)
+  udf_name="python${udf_suffix}"
+  python_name="python${version}"
+
+  python3 $YDB_SOURCE_PATH/ya make $YDB_SOURCE_PATH/ydb/library/yql/udfs/common/python/system_python/${udf_name} \
+    -T ${BUILD_FLAGS} --ignore-recurses -DSTRIP=yes -DOS_SDK=local -DUSE_ARCADIA_PYTHON=no -DUSE_LOCAL_PYTHON -DPYTHON_CONFIG=${python_name}-config -DPYTHON_BIN=${python_name} --output $YQL_BUILD_PATH
```

Comment:
`-DOS_SDK=local` here allows to avoid some bug in ya, when `/usr/include/pythonX/pyconfig.h` tries to include `ARCH/pythonX/pyconfig.h` and fails. It can be fixed some other way (by creating some symlinks and tricking it), but `-DOS_SDK=local` just works (cause we are in pure ubuntu focal docker with only pythons installed)

Reply:
I do not like this approach. OS_SDK local is intended to enable other functionality, and considering how little these options are documented it is unwise to use them "just because it works". For example, I am not sure that running this script in the same ubuntu release as the main docker image is what we need, maybe we would need to run it using the same sdk the ya make toolchain uses internally. 

Let's use one of the hacks without OS_SDK=local, and create a ticket for yatool about this bug, if we haven't already.